server/db/db.js

```diff
@@ -870,6 +870,32 @@ async function getDataForExport(clientParam) {
   }
 }
 
+async function getCurrentTime(clientParam) {
+  let client = clientParam
+  const transactionMode = typeof client !== 'undefined'
+
+  try {
+    if (!transactionMode) {
+      client = await pool.connect()
+    }
+
+    const { rows } = await client.query('SELECT NOW()')
```

Comment:
Did you mean to leave this as `NOW()`? Does it work on the server as well?

Reply:
Yes - this way I (or anyone else who doesn't have the same PSQL version as the server) don't need to change back and forth between `now` & `current_timestamp`, it's now across the board in both places now 😅